src/test/java/com/petstore/petstoreRest/service/UserServiceImplTest.java

```diff
@@ -35,19 +35,27 @@ public class UserServiceImplTest {
 
     private User user;
     private UserDTO userDTO;
+    private List<User> users;
     private List<UserDTO> userDTOs;
 
     @BeforeEach
     public void init() {
         user = new User(1L, "user", "first", "last",
```

Comment:
please extract logic for creating of test object to UserServiceTestFactory and reuse them in the test

Reply:
Changed